README.md

```diff
@@ -55,6 +55,21 @@ steps:
 - run: npm test
 ```
 
+**Caching pnpm dependencies:**
+```yaml
+steps:
+- uses: actions/checkout@v2
+- uses: pnpm/action-setup@v2
```

Comment:
Since this is an third party action not supported officially by us can you add this disclaimer at the top of the YAML snippet

```
# This workflow uses actions that are not certified by GitHub.
# They are provided by a third-party and are governed by
# separate terms of service, privacy policy, and support
# documentation.
```

Could you also pin to a specific SHA version instead of `v2` for the `pnpm` action. This is more for security so that in-case something is compromised with the third-party action than potential impact is reduced.

Here is a similar example that can be used as a reference: https://docs.github.com/en/actions/guides/building-and-testing-python#publishing-to-package-registries

Reply:
Updated, I've added the disclaimer and pinned to the latest commit on the `v2` branch 👍